### PR TITLE
feat: cli addition of starting cli - limited to key gen for now

### PR DESCRIPTION
**Pull request by mprorock**
Adds a typescript CLI for interacting with vc http api methods and common vc utlities such as key generation, etc

### CODE REVIEW

File: packages/cli/dist/index.js [hunk 1]
```diff
@@ -0,0 +1,67 @@
+#!/usr/bin/env node
```
Comment (OR13) [inline packages/cli/dist/index.js:1]: please git ignore your dist

File: packages/cli/package.json [hunk 2]
```diff
@@ -0,0 +1,41 @@
+{
+  "name": "vc-http-api-cli",
```
Comment (OR13) [inline packages/cli/package.json:2]: awesome, I can setup CD so this package gets published on merge

File: packages/cli/package.json [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+  "version": "1.0.0",
+  "description": "CLI for vc-http-api implementation",
+  "main": "src/index.js",
+  "bin": {
+    "vc-util": "dist/index.js"
+  },
+  "scripts": {
+    "test": "echo \"This CLI for the moment relies solely on the tests in the main services\" && exit 0",
```
Comment (OR13) [inline packages/cli/package.json:10]: sadge, no tests.

File: packages/cli/src/keys/keys.ts [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+      return [await ed25519.Ed25519KeyPair.generate(options)];
+    }
+    case "x25519": {
+      return [await x25519.X25519KeyPair.generate(options)];
+    }
+    case "bls12381": {
+      const keypairs = await bls12381.Bls12381KeyPairs.generate();
+      return [keypairs.g1KeyPair, keypairs.g2KeyPair];
+    }
+    case "p-256": {
```
Comment (OR13) [inline packages/cli/src/keys/keys.ts:25]: we might want to switch this to P-384 based on the SVIP Profile stuff, and yes, there are folks who only use P-384... in NZ...

**Review by OR13 — CHANGES_REQUESTED** [review]
only real blocker is the git ignore issue.

File: packages/cli/dist/index.js (hunk 1 above)
Comment (mprorock) [reply packages/cli/dist/index.js]: done

File: packages/cli/src/keys/keys.ts (hunk 4 above)
Comment (mprorock) [reply packages/cli/src/keys/keys.ts:25]: agreed - p384 can come later - basically stubbing in some stuff to match universal wallet, then adding credentials, then going back and expanding key types

**Review by OR13 — APPROVED** [review]

File: packages/cli/package.json (hunk 3 above)
Comment (mprorock) [reply packages/cli/package.json:10]: yep, will add once we have some more functionality to actually test - likely command line parameter validation with demo data

**Pull request merged**